ale_linters/chef/foodcritic.vim

```diff
@@ -30,10 +31,15 @@ function! ale_linters#chef#foodcritic#Handle(buffer, lines) abort
     return l:output
 endfunction
 
+" Support options!
+let g:ale_chef_foodcritic_options = get(g:, 'ale_chef_foodcritic_options', '')
+
 call ale#linter#Define('chef', {
 \   'name': 'foodcritic',
 \   'executable': 'foodcritic',
-\   'command': 'foodcritic %t',
+\   'command': 'foodcritic '
+\       . g:ale_chef_foodcritic_options
+\       . ' %t',
```

Comment:
While you're at it, could you replace `'command'` here with `'command_callback'` and a function which returns the string? I'm planning ahead for #427. I'll change all of the linters in the repository so they work like that eventually.